Known-Issues-With-Codepen.md

```diff
@@ -6,3 +6,9 @@ Free Code Camp learners are encouraged to use [Codepen.io](http://www.codepen.io
 2. **https plugin issue:** The Chrome extension "HTTPS Everywhere" (created by the [Electronic Frontiers Foundation](http://www.eff.org/)) has been known to interfere with AJAX calls. Because the extension automatically uses HTTPS, it can cause a "mixed content" error that prevents the JSON/XML data from being loaded. If you encounter such an error, make sure that your plugins are not the culprit. Optionally you could use [crossorigin.me](http://crossorigin.me) as a proxy.
 3. **imgur hotlinking:** If you use images from [http://imgur.com](http://imgur.com) they will not show up most of the time, this is due to their TOS. A way to solve this is to use an alternate service like [http://postimg.org](http://postimg.org)
 4. **auto reload:** By default, everytime you make changes in the HTML or JS editor windows, the preview window refreshes. You can turn this off and enable a "Run Button", by going to Settings > Behaviour > "Want a Run Button?" and unticking the box.
+5. **location.reload:** If you run into an issue of your code working in the debug view or in JSFiddle, but not in Codepen editor view or full page view, double check if you used `location.reload()`. If you did, you have to find another way to achive desired, because Codepen will strip `location.reload` and leave only `()` in your code. Read more [here:](https://blog.codepen.io/documentation/editor/things-we-strip/)
```

Comment:
it just breaks the whole program because it left orphan brackets in my code. shall I add location = location;
// OR
location.replace(location); as suggested solution. I didn't try that one myself